### PR TITLE
Docs on how to use Camera2D

### PR DESCRIPTION
**Pull request by ghost**
Adds a description of the most typical use case (for new users).

### CODE REVIEW

File: doc/classes/Camera2D.xml
Comment (akien-mga) [inline doc/classes/Camera2D.xml]: There should not be empty newlines in the XML. One line == one paragraph.

File: doc/classes/Camera2D.xml
Comment (akien-mga) [inline doc/classes/Camera2D.xml]: I don't think the description should start with "To use this". The description should explain what Camera2D is. Usage instructions should come at the end.

**Comment by akien-mga** [comment]
CC @godotengine/documentation

**Review by NathanLovato — CHANGES_REQUESTED** [review]
Please read the [writing guidelines](http://docs.godotengine.org/en/latest/community/contributing/docs_writing_guidelines.html) - can't merge a PR with a casual style, plus  see @akien-mga 's comments

**Comment by poke1024** [comment]
Let's leave this to the docs experts for sometime. I know how to use `Camera2D` (and I didn't learn it from the current docs).

**Comment by NathanLovato** [comment]
Not sure what you meant with your comment but you seem upset. Anyone's welcome to contribute, you just have to edit your PR to get it to merge because:
- If everyone writes in a different style (that was the case with Godot 2.x's reference) it makes the ref harder to browse and read
- Empty lines produce empty paragraphs, thus big gaps in the editor
- This is the code reference. It'd be great to have a good camera tutorial, or a page dedicated to the camera, but this goes in the [manual](https://github.com/godotengine/godot-docs/)

Everyone goes through review, me included, and there are often typos or things to improve before you get merged.

**Pull request closed without merging**
